bundles/org.openhab.binding.monopriceaudio/README.md

```diff
@@ -0,0 +1,149 @@
+```
+
+## Channels
+
+The following channels are available:
+
+| Channel ID                    | Item Type            | Description                                                                                                   |
+|-------------------------------|----------------------|---------------------------------------------------------------------------------------------------------------|
+| all:allon                     | Switch               | Turn all zones on simultaneously (except those specified by the ignoreZones config option)                    |
+| all:alloff                    | Switch               | Turn all zones off simultaneously (except those specified by the ignoreZones config option)                   |
+| all:allsource                 | Number               | Select the source input for all zones simultaneously (1-6) (except ignoreZones)                               |
+| all:allvolume                 | Dimmer               | Control the volume for all zones simultaneously (0-100%) [translates to 0-38] (except ignoreZones)            |
+| all:allmute                   | Switch               | Mute or unmute all zones simultaneously (except ignoreZones)                                                  |
+| zoneN#power (where N= 1-18)   | Switch               | Turn the power for a zone on or off                                                                           |
+| zoneN#source (where N= 1-18)  | Number               | Select the source input for a zone (1-6)                                                                      |
+| zoneN#volume (where N= 1-18)  | Dimmer               | Control the volume for a zone (0-100%) [translates to 0-38]                                                   |
+| zoneN#mute (where N= 1-18)    | Switch               | Mute or unmute a zone                                                                                         |
+| zoneN#treble (where N= 1-18)  | Number               | Adjust the treble control for a zone (-7 to 7) -7=none, 0=flat, 7=full                                        |
+| zoneN#bass (where N= 1-18)    | Number               | Adjust the bass control for a zone (-7 to 7) -7=none, 0=flat, 7=full                                          |
+| zoneN#balance (where N= 1-18) | Number               | Adjust the balance control for a zone (-10 to 10) -10=left, 0=center, 10=right                                |
+| zoneN#dnd (where N= 1-18)     | Switch               | Turn on or off the Do Not Disturb for the zone (for when the controller's external page trigger is activated) |
+| zoneN#page (where N= 1-18)    | Number               | Indicates if the page input is activated for the zone (ReadOnly 0-1)                                          |
+| zoneN#keypad (where N= 1-18)  | Number               | Indicates if the physical keypad is attached to a zone (ReadOnly 0-1)                                         |
```

Comment:
> 
> 
> If these have only two states, can you make them switches?

I can, but I guess I don't understand the desire for read-only switches. These are more like an indicator light. Does the UI render read-only switches appropriately? To me it is confusing to have a switch ui control that can't be turned on and off.

Reply:
I know what you mean. Maybe another reviewer has another opinion. Contact with open/closed could be an option, too.

Reply:
@fwolter 
I updated the PR to use a Contact instead of a Number. I set the state options to have a friendly description. But in the UI if I use a Switch, it shows up with.... a number! See screen shot. Any ideas how to correct that?

![image](https://user-images.githubusercontent.com/26532542/83456679-c095a800-a425-11ea-8c5c-739c9a028382.png)

Reply:
Why not using a Text type in the sitemap? Could be customized with a mapping transformation.